global.d.ts

```diff
@@ -0,0 +1,9 @@
+declare global {
+  namespace NodeJS {
+    interface ProcessEnv {
+      DB_URL: string;
```

Comment:
I don't think we need this as well. Types are for invariants that always hold true. We can't ensure `DB_URL` is always present, since it comes from external environment. Instead, to be more robust, we should explicitly check the presence of the environment variable at runtime and gives error if not present.

Reply:
makes sense!